sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Customized/Models/InstancesOperationErrorDetails.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// Additional error information.
```

Comment:
Do you think we can add some more info here? 
"Additional error information for any instance operation" or something more relevant.